### PR TITLE
Gas reporting

### PR DESCRIPTION
**Pull request by g-r-a-n-t**
### What was wrong?

We weren't tracking how much gas is used in our integration tests.

### How was it fixed?

We track it now.

### To-Do

[//]: # (Stay ahead of things, add list items here!)
- [ ] OPTIONAL: Update [Spec](https://github.com/ethereum/fe/blob/master/docs/src/spec/index.md) if applicable
- [ ] Add entry to the [release notes](https://github.com/ethereum/fe/blob/master/newsfragments/README.md) (may forgo for trivial changes)
- [x] Clean up commit history

### CODE REVIEW

File: crates/tests/src/snapshots/fe_compiler_tests__stress__external_calls_stress.snap [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
```
Comment (g-r-a-n-t) [inline crates/tests/src/snapshots/fe_compiler_tests__stress__external_calls_stress.snap:1]: delete

File: crates/tests/src/snapshots/fe_compiler_tests__features__revert.snap [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
```
Comment (g-r-a-n-t) [inline crates/tests/src/snapshots/fe_compiler_tests__features__revert.snap:1]: delete

File: crates/tests/src/snapshots/fe_compiler_tests__features__ctx_init_in_call.snap [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
```
Comment (g-r-a-n-t) [inline crates/tests/src/snapshots/fe_compiler_tests__features__ctx_init_in_call.snap:1]: delete

File: crates/tests/src/snapshots/fe_compiler_tests__features__assert.snap [hunk 4]
```diff
@@ -0,0 +1,6 @@
+---
```
Comment (g-r-a-n-t) [inline crates/tests/src/snapshots/fe_compiler_tests__features__assert.snap:1]: delete

File: crates/tests/src/snapshots/fe_compiler_tests__features__abi_decode_checks.snap [hunk 5]
```diff
@@ -0,0 +1,6 @@
+---
+source: crates/tests/src/features.rs
```
Comment (g-r-a-n-t) [inline crates/tests/src/snapshots/fe_compiler_tests__features__abi_decode_checks.snap:2]: delete

**Review by Y-Nak — APPROVED** [review]

**Pull request merged**